Source/WebCore/rendering/RenderThemeMac.mm

```diff
@@ -716,6 +716,11 @@ static Color activeButtonTextColor()
             return semanticColorFromNSColor(alternateColors[1]);
         }
 
+        case CSSValueAppleSystemTertiaryFill:
+            if ([NSColor respondsToSelector:@selector(tertiarySystemFillColor)])
+                return semanticColorFromNSColor([NSColor tertiarySystemFillColor]);
+            return semanticColorFromNSColor([NSColor colorWithGenericGamma22White:0.0 alpha:0.05]);
```

Comment:
Why does this one need to bounce through NSColor? Can it be more like the others above?